lib/linear_gauge/pointers/linear_gauge_pointer.dart

```diff
@@ -129,6 +144,15 @@ class LinearGaugeIndicator {
     }
   }
 
+  //
```

Comment:
There is no comment added 
If you wish to add the comment please do it or else remove the `//`